Aliucord/src/main/java/com/aliucord/settings/Crashes.java

```diff
@@ -64,14 +66,63 @@ public Context getContext() {
     @SuppressLint("SetTextI18n")
     public void onViewBound(View view) {
         super.onViewBound(view);
-
         setActionBarTitle("Crash Logs");
 
         Context context = requireContext();
-        LinearLayout layout = (LinearLayout) ((NestedScrollView) ((CoordinatorLayout) view).getChildAt(1)).getChildAt(0);
+        int padding = Utils.getDefaultPadding();
+        int p = padding / 2;
+        File folder = new File(Constants.BASE_PATH, "crashlogs");
+        File[] files = folder.listFiles();
```

Comment:
for performance reasons this should be moved into the clearLogsBtn onClick handler

Reply:
These values get used in both click handlers AND outside the click handlers for setting the clear logs opacity, if you'd like them moved please suggest something that can accommodate this.

Reply:
true nevermind